.github/workflows/ci.yml

```diff
@@ -51,27 +51,13 @@ jobs:
     strategy:
       matrix:
         include:
-            - os: ubuntu-20.04
-              python-version: 3.6
-              toxenv: py36-fuse2
-            - os: ubuntu-20.04
+            - os: windows-latest
               python-version: 3.7
-              toxenv: py37-fuse2
-            - os: ubuntu-20.04
-              python-version: 3.8
-              toxenv: py38-fuse3
-            - os: ubuntu-20.04
-              python-version: 3.9
-              toxenv: py39-fuse3
-            - os: macos-latest
-              # note: it seems that 3.8 and 3.9 are currently broken,
-              # neverending RuntimeError crashes...
-              python-version: 3.7
-              toxenv: py37-fuse2
+              toxenv: py37-none
 
     env:
       # Configure pkg-config to use OpenSSL from Homebrew
-      PKG_CONFIG_PATH: /usr/local/opt/openssl@1.1/lib/pkgconfig
+      PKG_CONFIG_PATH: C:\Program Files\OpenSSL-Win64
```

Comment:
just a placeholder. i have no idea where it is preinstalled, nor whether it supports pkg-config.

Reply:
Obviously, see CI run, this is the wrong place and should be fixed.

Preferably:
1. fix this, so pkg-config finds the openssl libs/headers of the preinstalled openssl
2. if pkg-config can't work (e.g. because the preinstalled openssl does not support it), use BORG_OPENSSL_PREFIX to point to the preinstalled openssl
3. only install an own version of openssl if absolutely necessary (slow!)